packages/osd-test/src/functional_test_runner/integration/basic.test.ts

```diff
@@ -38,11 +38,9 @@ const SCRIPT = resolve(REPO_ROOT, 'scripts/functional_test_runner.js');
 const BASIC_CONFIG = require.resolve('../fixtures/simple_project/config.js');
 
 describe('basic config file with a single app and test', function () {
-  this.timeout(60 * 1000);
```

Comment:
Why are we removing the timeouts?

Reply:
My understanding is that Jest handles timeouts as config values, not something that's specified in code for a specific suite the way mocha does. But let me know if there's an equivalent jest syntax for this that I missed.

Reply:
I see functional test runner and I think this is selenium right?

Reply:
This was actually a mocha test within the functional test runner - my understanding is that it's purpose is to validate that the functional test runner itself can start up correctly. So now this test will be run by jest instead.

Reply:
<img width="805" alt="Screen Shot 2022-05-26 at 12 34 50 PM" src="https://user-images.githubusercontent.com/1679762/170563737-34e0e35f-b527-46af-80ce-d19920b8be4f.png">

Given that the jest suites for this package run in a couple seconds, I think it's fine.